tests/testthat/test-content.R

```diff
@@ -231,4 +231,3 @@ with_mock_api({
     expect_identical(v$key, "WrEKKa77")
   })
 })
-
```

Comment:
Errant whitespace change her too, yeah?

Reply:
I think there was a double-new-line at the end of this file, and perhaps it was fixed automatically by my editor at some point, or perhaps I fixed it. Either way, the whitespace at the end of the file is correct as it stands.